wpeverest/everest-forms
# EVF-230 Fix - WP nonce issue with multiple form on same page

### All Submissions:






### Changes proposed in this Pull Request:



Duplicate CSS ID issue with wp nonce when including multiple forms on same page.

### How to test the changes in this Pull Request:

1. Create Form
2. Add two different forms in same page. 
3. Submit Entries.
4. Test Email, PDF, Payment, Captcha ETC.
5. Verify its working as expected.

### Types of changes:




### Other information:




### Changelog entry

> Fix - WP nonce issue with multiple form on same page.
